src/coreclr/src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -14,31 +14,31 @@ public class CustomAttributeData
         #region Public Static Members
         public static IList<CustomAttributeData> GetCustomAttributes(MemberInfo target)
         {
-            if (target == null)
+            if (target is null)
```

Comment:
I assume these changes from `==` to `is` prevents the more expensive call to the == operator on `Type`?

Reply:
Yes